Project-Hotpot/Project-Hotpot/Parse-Backend/PFPlaylist.swift

```diff
@@ -155,6 +153,39 @@ class PFPlaylist: PFObject, PFSubclassing {
             }
         }
     }
+    
+    class func removePlaylistInBackground(playlist:PFPlaylist, completion: @escaping (_ result: Result<Void, Error>) -> Void) {
+        
+        //get all songs from given playlist
+        getAllSongsFromPlaylist(playlist: playlist) { result in
+            switch result {
+            case .success(let songArray):
+                for song in songArray {
+                    //deletes song from join table
+                    removeSongFromPlaylistInBackground(song: song, playlist: playlist) { result in
+                        switch result {
+                        case .success(_):
+                            NSLog("Deleted song successfully")
+                            
+                        case .failure(let error):
+                            NSLog("error deleting song from playlist: \(error)")
+                            return completion(.failure(error))
+                        }
+                    }
+                }
```

Comment:
Hmmm this feels a bit fishy that we have to manually break song relation with playlist before deleting. I would imagine that Parse would internally do this when i call delete on a playlist object? Could you double check this?